app/javascript/mastodon/features/ui/index.js

```diff
@@ -361,10 +361,9 @@ class UI extends React.PureComponent {
       this.props.dispatch(closeOnboarding());
     }
 
-    this.props.dispatch(fetchMarkers());
     this.props.dispatch(expandHomeTimeline());
     this.props.dispatch(expandNotifications());
-
+    setTimeout(() => this.props.dispatch(fetchMarkers()), 500);
```

Comment:
I just felt there were too many requests issued at exactly the same time

500ms is barely noticable for a user but may reduce load on the server

Reply:
if anything i think it would be best if we ensured this request was done before the notifications one, especially on the advanced layout, in order to avoid the unread markers popping “late”